.github/workflows/swift-toolchain.yml

```diff
@@ -1020,16 +1042,21 @@ jobs:
                 -D SWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE=${{ github.workspace }}/SourceCache/swift-syntax `
                 -D SWIFT_PATH_TO_STRING_PROCESSING_SOURCE=${{ github.workspace }}/SourceCache/swift-experimental-string-processing
       - name: Build Swift Standard Library
-        run: cmake --build ${{ github.workspace }}/BinaryCache/swift
+        run: |
+          Remove-Item env:\SDKROOT
```

Comment:
I wonder if we can write `SDKROOT=` to `GITHUB_ENV` to remove the variable once and for all?
(in a future change)